src/User32.Desktop/User32.cs

```diff
@@ -57,7 +56,61 @@ public static partial class User32
         /// <summary>Windows icon or the icon of the window specified in <see cref="MENUITEMINFO.dwItemData" />.</summary>
         public static readonly IntPtr HBMMENU_SYSTEM = new IntPtr(1);
 
-        public delegate int WindowsHookDelegate(int code, IntPtr wParam, IntPtr lParam);
+        /// <summary>
+        /// An application-defined or library-defined callback function used with the <see cref="SetWindowsHookEx(WindowsHookType, IntPtr, IntPtr, int)"/> function.
+        /// This is a generic function to Hook callbacks. For specific callback functions see CallWndDelegate, CallWndRetDelegate, CBTDelegate, DebugDelegate, ForegroundIdleDelegate,
+        /// GetMsgDelegate, JournalPlaybackDelegate, JournalRecordDelegate, KeyboardDelegate, LowLevelKeyboardDelegate, MessageDelegate, MouseDelegate, LowLevelMouseDelegate,
+        /// ShellDelegate and SysMsgDelegate.
+        /// </summary>
+        /// <param name="nCode">An action code for the callback. Can be used to indicate if the hook procedure must process the message or not.</param>
+        /// <param name="wParam">First message parameter</param>
+        /// <param name="lParam">Second message paramater</param>
+        /// <returns>
+        /// An LRESULT. Ususally if nCode is less than zero, the hook procedure must return the value returned by CallNextHookEx.
+        /// If nCode is greater than or equal to zero, it is highly recommended that you call CallNextHookEx and return the value it returns;
+        /// otherwise, other applications that have installed hooks will not receive hook notifications and may behave incorrectly as a result.
+        /// If the hook procedure does not call CallNextHookEx, the return value should be zero.
+        /// </returns>
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        public delegate int WindowsHookDelegate(int nCode, IntPtr wParam, IntPtr lParam);
+
+        /// <summary>
+        /// An application-defined callback function used with the <see cref="EnumWindowStations(EnumWindowStationDelegate, IntPtr)"/> function.
+        /// </summary>
+        /// <param name="lpszWindowStation">The name of the window station.</param>
+        /// <param name="lParam">An application-defined value specified in the <see cref="EnumWindowStations(EnumWindowStationDelegate, IntPtr)"/> function.</param>
+        /// <returns>To continue enumeration, the callback function must return TRUE (non-zero value). To stop enumeration, it must return FALSE (0).</returns>
+        /// <remarks>
+        /// An application must register this callback function by passing its address to <see cref="EnumWindowStations(EnumWindowStationDelegate, IntPtr)"/>.
+        /// The callback function can call SetLastError to set an error code for the caller to retrieve by calling GetLastError.
+        /// </remarks>
+        [UnmanagedFunctionPointer(CallingConvention.StdCall, CharSet = CharSet.Ansi)]
```

Comment:
Why do you set `CharSet.Ansi` instead of Unicode here? The native definition appears to be `LPTSTR` so I expect on all the supported operating systems it will always be Unicode, won't it?